lib/rialto/etl/configs/stanford_grants_to_sparql_statements.rb

```diff
@@ -48,25 +48,30 @@
 
 # See https://wiki.duraspace.org/display/VTDA/VIVO-ISF+1.6+relationship+diagrams%3A+Grant
 to_field VIVO.relates.to_s, lambda { |json, accum|
-  person = RIALTO_PEOPLE[json['piEmployeeId']]
-  pi_role = RIALTO_CONTEXT_ROLES["#{json['spoNumber']}_#{json['piEmployeeId']}"]
+  family_name, given_name = json.fetch('piFullName', '').split(', ')
+  # Use `#to_s` explicitly here in case piFullName isn't a name string like we expect
+  person = Rialto::Etl::Transformers::People.resolve_or_construct_person(given_name: given_name.to_s,
```

Comment:
Somewhere below you need to add `person` instead of `person_uri`. Otherwise, a new person will not be added -- you'll just use the URI.

Reply:
OK, got it. Pushing momentarily.

Reply:
I think we need to pull in @peetucket to sound off on this:

If a grant PI's SUNet ID is not found in the researcher data (from profiles system)—perhaps in the instance of a faculty member who left the university a year ago?—should we create a person entity for that person so we can tie the grant to a (former?) Stanford person? Or should we just load the grant without trying to link it to a person? (We can still display the PI's name and/or SUNet ID without linking it, if valuable.)